src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Tooltip/DefaultVSLSPTagHelperTooltipFactory.cs

```diff
@@ -415,9 +415,19 @@ static void ClassifyExistingTextRun(List<ClassifiedTextRun> runs, StringBuilder
 
     private static ContainerElement CombineClassifiedTextRuns(IReadOnlyList<DescriptionClassification> descriptionClassifications, ImageElement glyph)
     {
+        var isFirstElement = true;
         var classifiedElementContainer = new List<ContainerElement>();
         foreach (var classification in descriptionClassifications)
         {
+            if (isFirstElement)
+            {
+                isFirstElement = false;
+            }
+            else
+            {
+                classifiedElementContainer.Add(new ContainerElement(ContainerElementStyle.Wrapped, new ClassifiedTextElement()));
```

Comment:
It would be nice if we could get a separation line back, but from a quick look at the legacy code I'd guess it may not be compatible with LSP. Also the accessibility on the old separator is TERRIBLE in dark mode. I've got perfect vision (when corrected) and I almost don't notice that it exists, so I feel like it's not a huge deal if we don't get the separator back.